src/NuProj.Tests/packages.config

```diff
@@ -1,15 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Web.Xdt" version="2.1.1" targetFramework="net451" />
-  <package id="Nerdbank.GitVersioning" version="1.1.52-rc" targetFramework="net451" developmentDependency="true" />
-  <package id="NuGet.CommandLine" version="2.8.6" />
-  <package id="NuGet.Core" version="2.8.6" targetFramework="net451" />
-  <package id="System.Collections.Immutable" version="1.1.36" targetFramework="net451" />
-  <package id="xunit" version="2.0.0" targetFramework="net451" />
+  <package id="Nerdbank.GitVersioning" version="1.2.1" targetFramework="net451" developmentDependency="true" />
+  <package id="NuGet.CommandLine" version="3.3.0" targetFramework="net451" />
```

Comment:
I thought I'd tried updating the NuGet.CommandLine package a while back and couldn't because newer versions dropped some of the public API we called into from the unit tests. But I might be confusing this project with another. Can you confirm that you've run all unit tests and that they pass?